schema.development.kf

```diff
@@ -354,6 +395,8 @@ procedure remove_credential($id uuid) public {
     AND human_id=(SELECT DISTINCT human_id FROM wallets WHERE (wallet_type = 'EVM' AND address=@caller COLLATE NOCASE)
         OR (wallet_type = 'NEAR' AND public_key = @caller)
     );
+
+    DELETE FROM internal_access_grants WHERE data_id = $id;
```

Comment:
We should follow up on this.

Every time that a credential is inserted, we should call `define_ag_status_as_owner` (or an adaptation of that) to make sure the `status` is up-to-date.

None of the current integration flows are affected by this "hole", but we should keep our state in a non-surprising state.